bigtable/tests/run_integration_tests.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+function kill_emulator {
+    kill ${EMULATOR_PID}
+    wait >/dev/null 2>&1
+}
+
+echo "Launching Cloud Bigtable emulator in the background"
+"${GOPATH}/bin/emulator" >emulator.log 2>&1 </dev/null &
+EMULATOR_PID=$!
+if [ $? -ne 0 ]; then
+    echo "emulator failed, aborting"
+    cat emulator.log
+    exit 1
+fi
+
+trap kill_emulator EXIT
+
+export BIGTABLE_EMULATOR_HOST=localhost:9000
```

Comment:
Is `9000` the default port on which the emulator listens on? Is this running inside a container so we're guaranteed that 9000 is not taken already? Is it worthwhile to document where this magic number comes from? Is it worthwhile to choose a specific port explicitly and use a var so the connection is clear?

Reply:
Done.